src/main/java/network/brightspots/rcv/ResultsWriter.java

```diff
@@ -95,11 +95,11 @@ static String getOutputFilePath(
       String outputDirectory,
       String outputType,
       String timestampString,
-      Integer sequentialTabulationNumber) {
+      String sequentialTabulationId) {
```

Comment:
I'm curious why you bothered to change this. Seems unrelated to your other changes.

Reply:
When I refactored contest Id to a string (was Integer) this variable type and name needed to change to match the new type.